hack/mirror-istioctl.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Usage
+# ./sync-istioctl.sh 1.9.0,1.10.1
+
+set -e
+
+LOCAL_ISTIOCTL_DIR=istioctl-mirror
+
+mkdir -p $LOCAL_ISTIOCTL_DIR
+
+(
+    cd $LOCAL_ISTIOCTL_DIR
+
+    # Loop over comma-separated list of istio versions
+    for i in $(echo $1 | sed "s/,/ /g")
```

Comment:
You refer to $1 here, shouldn't it be `ISTIO_VERSIONS` instead?

Reply:
You're right, thanks!